site/content/en/blog/releases/1.20.0-rc.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "1.20.0-rc - simple game server version to 0.6"
+linkTitle: "1.20.0-rc"
+date: "2022-01-04"
+---
+
+
+This is the 1.20.0-rc release of Agones.
+
+This release brings us  the simple game server version 0.6 and grpc client and server on same version
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" >README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+Update the simple game server version to 0.6.[ #2413](https://github.com/googleforgames/agones/pull/2413)
+Update the minikube driver flag name (--vm-driver is deprecated) [#2399]([url](https://github.com/googleforgames/agones/pull/2399))
+Unreal SDK misses "const" prefix for function parameters. [#2411](https://github.com/googleforgames/agones/issues/2411)
+TLS handshake error with multi-cluster setup [#2402](https://github.com/googleforgames/agones/issues/2402)
+Upgrade-gRPC-version [ #2414](https://github.com/googleforgames/agones/pull/2414)
+Update the simple game server to move itself back to the Ready state after allocation [#2409](https://github.com/googleforgames/agones/pull/2409)
+
+Documentation: https://development.agones.dev/site/
+
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.20.0-rc/CHANGELOG.md" >CHANGELOG</a> for more details on changes.
```

Comment:
This will need data-proofer-ignore - see the template I linked in the last comment. (Since the branch doesn't exist yet).